ledger/metrics/src/test/suite/scala/com/daml/telemetry/TelemetrySpec.scala

```diff
@@ -96,5 +117,38 @@ class TelemetrySpec
           attributes should contain(aCommandIdSpanAttribute)
         }
     }
+
+    "record an exception" in {
+      TestTelemetry
+        .runFutureInSpan(
+          aSpanName,
+          SpanKind.Internal,
+          anApplicationIdSpanAttribute,
+        ) { _ =>
+          Future.failed(anException)
+        }
+        .recover { case _ =>
+          val spanAttributes = spanExporter.finishedSpanAttributes
+          spanAttributes should contain(anApplicationIdSpanAttribute)
+
+          assertThrown
+        }
+    }
   }
+
+  private def assertThrown: Assertion = {
```

Comment:
You might want to rename this to `assertExceptionInEventAttributes`/`assertExceptionRecorded` or similar, as by reading the test case you can easily mix this with or misread this as [assertThrows](http://doc.scalatest.org/3.0.1-2.12/org/scalatest/Assertions.html#assertThrows[T%3C:AnyRef](f:=%3EAny)(implicitclassTag:scala.reflect.ClassTag[T],implicitpos:org.scalactic.source.Position):org.scalatest.Assertion).